lib/helpers.ts

```diff
@@ -1,6 +1,7 @@
 import _ from 'lodash';
 import { NextRouter } from 'next/router';
 import fetch from 'node-fetch';
+import pathParse from 'path-parse';
```

Comment:
feels slightly overkill to include a path parser dependency to do `lastIndexOf('/')`? No big deal tho, maybe it'll be useful in the future

Reply:
good point. actually, there is also builtin node path module, probably better to use them instead of this one.

Reply:
removed the `path-parse` dependency, using builtin node path package now.